pkg/skaffold/initializer/init.go

```diff
@@ -54,6 +54,18 @@ type Initializer interface {
 	GetImages() []string
 }
 
+// InitBuilder represents a builder that can be chosen by skaffold init.
+type InitBuilder interface {
+	// GetPrompt returns the initBuilder's string representation, used when prompting the user to choose a builder.
+	GetPrompt() string
+	// GetArtifact returns the Artifact used to generate the Build Config.
+	GetArtifact(image string) *latest.Artifact
+	// GetConfiguredImage returns the target image configured by the builder
+	GetConfiguredImage() string
```

Comment:
Disconnect between `ConfiguredImage` and `target image`?  is it ok to have an empty string, and what does it mean?